src/main/java/com/marklogic/mgmt/admin/AdminManager.java

```diff
@@ -323,4 +330,31 @@ public RestTemplate getRestTemplate() {
 		}
 		return this.restTemplate;
 	}
+
+	private ImmutablePair<Boolean, Boolean> checkCredentialsAndReplaceNulls(AdminConfig adminConfig) {
+		boolean setNullUsername = false;
+		boolean setNullPassword = false;
+		if (adminConfig.getUsername() == null) {
+			adminConfig.setUsername("");
+			setNullUsername = true;
+			AdminManager.this.restTemplate = null;
```

Comment:
You can just do `this.restTemplate` - I was not aware that this syntax actually worked. For consistency, remove "AdminManager" here and just do `this` (I ran the tests to verify it still works).

Reply:
The "AdminManager" part was required when the line was in the anonymous function.